src/spec-node/devContainers.ts

```diff
@@ -146,6 +147,7 @@ export async function createDockerParams(options: ProvisionOptions, disposables:
 		secretsP,
 		buildxPlatform: options.buildxPlatform,
 		buildxPush: options.buildxPush,
+		buildxLabel: options.buildxLabel,
```

Comment:
I am also seeing many failures, can you fix those? 👇 

![image](https://github.com/user-attachments/assets/c8d6b7f5-e547-4778-a629-7c7a1f2b3ece)

Reply:
fixed all the test cases